### DISCUSSION TITLE
[Config] mount Flashdrive with config file

### DISCUSSION THREAD

**Discussion by thomas-repo** [Ralim/IronOS#1260] [Ideas]
Hi,
i would like to ask if there are any plans to make a flashdrive available alike the original firmware does? 

In order to be able to tweak things like SLEW_RATE and min/max temperatures on the fly.

I know it's alot of work to implement the flashmount and USB, so im just asking if someone already joined the boat yet?

Thanks 
Thomas

**Comment by Ralim** [comment]
Hello,

All settings are shown in the on device settings menu.
If you want to have extra settings you think are missing open an issue to request them :)


There are no plans to add USB configuration support as it requires a lot of work and serves no "real" purpose that isn't covered by the settings menu.
